itest/mint_fund_seal_test.go

```diff
@@ -577,6 +577,9 @@ func testMintExternalGroupKeyChantools(t *harnessTest) {
 	mintReq2 := CopyRequest(issuableAssets[0])
 	mintReq2.Asset.Name = "itestbuxx-money-printer-brrr-tranche-2"
 	mintReq2.Asset.ExternalGroupKey = externalGroupKey
+	mintReq2.Asset.GroupedAsset = true
+	mintReq2.Asset.NewGroupedAsset = false
+	mintReq2.Asset.GroupKey = batchAssets[0].AssetGroup.TweakedGroupKey
```

Comment:
Why is `mintReq2.Asset.GroupedAsset = true` necessary here? I would have expected setting `mintReq2.Asset.ExternalGroupKey` to imply group asset sufficiently.

Is setting these fields necessary or are we just covering all possibilities here?

Reply:
The RPC (and by extension the CLI) interface for the minting call is not optimal and quite confusing. See https://github.com/lightninglabs/taproot-assets/pull/1517#issuecomment-2859009609

So you need to set at least one of the `GroupedAsset` or `NewGroupedAsset` to `true` if you want to start or issue into a group, independent of the other fields.